test/runner.js

```diff
@@ -30,6 +31,7 @@ if ( program.grep ) {
 mocha.suite.beforeAll( function() {
 	Chai.use( immutableChai );
 	Chai.use( sinonChai );
+	sinon.assert.expose( Chai.assert, { prefix: '' } );
```

Comment:
My second thought is that `sinon-chai` (http://sinonjs.org/docs/#assertions) in most cases duplicates what `sinon.assert.expose` offers (https://github.com/domenic/sinon-chai/blob/master/lib/sinon-chai.js#L117) ...

We should figure out if it makes sense to keep both approaches or recommend in docs one of them :)